examples/bun-bakery/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "version": "0.1.8",
```

Comment:
The version doesn't need to be synced to the ``bun-bakery`` version. Instead, please use ``1.0.0``.